### DISCUSSION TITLE
Is Uno currently supported in MAC and Linux ?

### DISCUSSION THREAD

**Discussion by ayankumaar** [unoplatform/uno#5036] [Q&A] [answered]
Is Uno currently supported in MAC and Linux ? In other words can the same desktop app developed in windows using uno will also run in MAC and linux ?

**Though here https://platform.uno/docs/articles/faq.html it says uno is not currently supported in MAC and linux. But, here https://platform.uno/ it says that it is supported in MAC and Linux as well. An answer on this will make me understand it better.**

Also, I would like to know is it necessary to install **mono** to develop desktop applications in uno ?or only .NET Core is enough ? I am planning to use VS Code as an IDE for the development.

**Comment by davidjohnoliver** [comment, marked as answer]
That FAQ is out of date - we'll update it soon. Mac and Linux are indeed currently supported.

You can check (up to date) instructions for setting up an Uno environment here:
VS Code: https://platform.uno/docs/articles/get-started-vscode.html
Linux-specific steps: https://platform.uno/docs/articles/get-started-with-linux.html

<blockquote><img src="../images/UnoLogoSmall.png" width="48" align="right"><div><strong><a href="https://platform.uno/docs/articles/get-started-vscode.html">Getting Started on VS Code</a></strong></div></blockquote>
<blockquote><img src="../images/UnoLogoSmall.png" width="48" align="right"><div><strong><a href="https://platform.uno/docs/articles/get-started-with-linux.html">Getting started with Uno Platform support for Linux</a></strong></div></blockquote>
